curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md

```diff
@@ -8,7 +8,14 @@ dashedName: state-name-puzzle
 
 # --description--
 
-This task is inspired by Mark Nelson's DDJ Column "Wordplay" and one of the weekly puzzle challenges from Will Shortz on NPR Weekend Edition [\[1\]](https://www.npr.org/templates/story/story.php?storyId=9264290) and originally attributed to David Edelheit. The challenge was to take the names of two U.S. States, mix them all together, then rearrange the letters to form the names of two *different* U.S. States (so that all four state names differ from one another). What states are these? The problem was reissued on [the Unicon Discussion Web](https://tapestry.tucson.az.us/twiki/bin/view/Main/StateNamesPuzzle) which includes several solutions with analysis. Several techniques may be helpful and you may wish to refer to [Gödel numbering](https://en.wikipedia.org/wiki/Goedel_numbering), [equivalence relations](https://en.wikipedia.org/wiki/Equivalence_relation), and [equivalence classes](https://en.wikipedia.org/wiki/Equivalence_classes). The basic merits of these were discussed in the Unicon Discussion Web. A second challenge in the form of a set of fictitious new states was also presented.
+**State name** is a <em>wordplay</em> in which you take the letters from two different state names and rearrange them to create two new state names.
+
+Example:
+
+```js
+const puzzleArray= ["North Carolina ", "South Dakota"];
+const solvedArray= ["North Dakota", "South Carolina"];
+```
```

Comment:
I think this could be removed. It does illustrate the point well, but gives away one of the answers we're looking for.

```suggestion
```